user-data.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+yum update -y
+yum install -y httpd
+systemctl start httpd && systemctl enable httpd
```

Comment:
could you please, add a newline at the end of file.

Reply:
Will do that now, thanks